### PR TITLE
add nosplashtexts pack

### PR DESCRIPTION
**Pull request by RayDeeUx**
is category 5 legal?

edit: this question is no longer worth discussing as kti has notified me that omitting categories is also valid

### CODE REVIEW

**Comment by KTibow** [comment]
splash text ideas:
- imagine making a pr for a suggestion that hasn't been `/suggestion accept`ed
- Don't forget to either upload your pack to the repo or set the `url`, otherwise it won't work!

**Comment by KTibow** [comment]
i only now realize you named this branch patch-69 -_-

**Comment by RayDeeUx** [comment]
他妈的

File: files/packs.json [hunk 1]
```diff
@@ -684,6 +684,24 @@
         "file": "SkyMobs.zip",
         "hash": "f226121d849d529fd2c23c47dd9b552b"
     },
+    {
+        "id": "nosplashtexts",
+        "display": "NoSplashTexts",
+        "creator": "BlockyRick, multiverison port by Erymanthus",
+        "description": "Multiversion resource pack to remove title screen splash text.",
```
Comment (KTibow) [inline files/packs.json]: Why do we care if it's multiversion

File: files/packs.json [hunk 2]
```diff
@@ -684,6 +684,24 @@
...
+        "icon": "nosplashtexts.png",
+        "icon_scaling": "pixel",
+        "actions": [
+            {
+                "text": "Github",
+                "link": "https://github.com/RayDeeUx/NoSplashTexts/releases/download/v1.0.0/NoSplashTexts.zip"
```
Comment (KTibow) [inline files/packs.json]: Why is the GitHub a direct download link instead of the project page

**Review by KTibow — CHANGES_REQUESTED** [review]

File: files/packs.json (hunk 2 above)
Comment (RayDeeUx) [reply files/packs.json:697]: gah!

File: files/packs.json (hunk 1 above)
Comment (RayDeeUx) [reply files/packs.json:691]: in the incredible offchance that skyclient shifts versions bc admins decided "yes lets just do the skyblock 1.17 swap"

**Comment by KTibow** [comment]
`(E|A) for Effort`

**Comment by RayDeeUx** [comment]
> `(E|A) for Effort`

wtfdym

**Comment by KTibow** [comment]
https://regex101.com

**Pull request closed without merging**